docs/lib/datastore/fragments/native_common/setup-auth-rules.md

```diff
@@ -123,5 +123,50 @@ type YourModel @model @auth(rules: [{ allow: groups
   ...
 }
 ```
+## Configure Multiple Authorization Types
 
-<inline-fragment platform="js" src="~/lib/datastore/fragments/js/setup-auth-rules/multi-auth.md"></inline-fragment>
\ No newline at end of file
+For some use cases, you will want DataStore to use multiple authorization types. For example, an app might use `API Key` for public content and `Cognito User Pool` for personalized content once the user logs in.
+
+By default, DataStore uses your API's default authorization type specified in the `amplifyconfiguration.json` file. To change the default authorization type, run `amplify update api`. Every network request sent through DataStore uses that authorization type, regardless of the model's `@auth` rule. 
+
+To enable DataStore to use multiple authorization types based on the model's `@auth` rules, set the `authModeStrategy` property when creating the DataStore plugin:
```

Comment:
maybe _"set the `authModeStrategy` property when initializing DataStore"_ so it works for both native platforms? In iOS `authModeStrategy` is a property of `DataStoreConfiguration`

Reply:
```suggestion
To enable DataStore to use multiple authorization types based on the model's `@auth` rules, set the authModeStrategy property when initializing DataStore:
```